distributions/openhab/src/main/resources/bin/backup.bat

```diff
@@ -0,0 +1,19 @@
+@ECHO off
+SETLOCAL
+
+powershell -command "& { . .\backup.ps1; Backup-openHAB -OHDirectory %~1 -OHBackups %~2 -FileName %~3 }"
```

Comment:
To allow for RC - RAA. You can pretty up the arguments with variables if you want. Checks if arguments are empty. If you don't do this, PS throws an error about OHDirectory.

if [%1]==[] ( powershell -command "&{. .\runtime\bin\backup.ps1; Backup-openHAB }" ) else ( powershell -command "&{. .\runtime\bin\backup.ps1; Backup-openHAB -OHDirectory %~1 -OHBackups %~2 -FileName %~3 }" )